db/utils.go

```diff
@@ -97,6 +99,31 @@ func InitSQLDB(port int, host, user, password, name string) (*sqlx.DB, error) {
 	return db, nil
 }
 
+// InitTestSQLDB opens test PostgreSQL database
```

Comment:
I am afraid if we have another method to connect to the database it can break the idea of unit test because the tests will not use the same init database connection. What about to make the just one point of database connection with values comming from system environment?

Reply:
Please look at
https://github.com/hermeznetwork/hermez-node/pull/733/files#diff-82bdf3f3f39ca350a5ba8e048300773b62463fa65153367662399b11b9669059R124

InitSQLDB is called as before.
I made a new function only to avoid code bloat because there are too many places where the test database is opened.

Reply:
I see. It's only a wrapper to the regular InitSQLDB method. Thanks for the clarification.